style/index.css

```diff
@@ -0,0 +1,76 @@
+.delete-btn{
+    -webkit-transition-duration: 0.4s; /* Safari */
+    transition-duration: 0.4s;
+}
+
+.delete-btn:hover {
+    background-color: #FF0000; /* Green */
+    color: white;
+}
+
+.list-element{
+    display : flex;
+    vertical-align: middle;
+}
+
+.list-element{
+    margin: 10px 0 0 0;
+}
+
+#task-description, #todo-list {
+    margin: 30px 0 0 0;
+}
+
+/* Dropdown Button */
+.dropbtn {
+    background-color: #4CAF50;
+    color: white;
+    padding: 16px;
+    font-size: 16px;
+    border: none;
+}
+
+/* The container <div> - needed to position the dropdown content */
+.dropdown {
+    position: relative;
+    display: inline-block;
+}
+
+/* Dropdown Content (Hidden by Default) */
+.dropdown-content {
+    position: absolute;
+    background-color: #f1f1f1;
+    min-width: 160px;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+}
+
+
+/* Links inside the dropdown */
+.dropdown-content a {
+    color: black;
+    padding: 12px 16px;
+    text-decoration: none;
+    display: block;
+}
+
+/* Change color of dropdown links on hover */
+.dropdown-content a:hover {
```

Comment:
Not clear, can't close the dropdown on click of a button.
Should be a real HTML element